### PR TITLE
fix(ME-2654): fix command execution with pty for connector built-in ssh server (ansible fix)

### PR DESCRIPTION
**Pull request by btoonk**
# Description

fix command execution with pty for connector built-in ssh server (ansible fix)

Fixes # (ME-2654)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Tested with:
* Ansible
* ansible without ControlMaster
* ansible with pipelining
* echo "ls -l" | ssh -tt ..
*  ssh -tt .. ls -l
*  ssh -tt .. wrongcommand
*  ssh with normal session

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: internal/ssh/server/server.go [hunk 1]
```diff
@@ -188,6 +189,6 @@ func GetShell(user *user.User) (string, error) {
 
 func execCmd(s ssh.Session, cmd exec.Cmd, uid, gid uint64, username string) {
 	pty, winCh, isPty := s.Pty()
-	exitCode := ExecCmd(s, s.RawCommand(), pty.Term, isPty, winCh, cmd, uid, gid, username)
+	exitCode := ExecCmd(context.TODO(), s, s.RawCommand(), pty.Term, isPty, winCh, cmd, uid, gid, username)
```
Comment (adrianosela) [inline internal/ssh/server/server.go:192]: the ssh session has a context object (`s.Context()` we could use here).

**Review by adrianosela — APPROVED** [review]
looks good, but one comment left.

**Review by th3wingman — APPROVED** [review]
tested manually, looks good! :ship:  !

File: internal/ssh/server/server.go (hunk 1 above)
Comment (btoonk) [reply internal/ssh/server/server.go:192]: This is the old ssh server, not the one you get with e2ee enabled.
But we can use the connection context, I will change that.

**Pull request merged**